revelations/perspire-test/f3d306d9-79ac-4087-afbc-46f739226eb2.json

```diff
@@ -0,0 +1 @@
9f5aa08a17cc13c1d5a457917195a4a4d146824225bb846178032178c629e9afeddcd62385582900a6127fe770b39607785aed0db1f951202fcc"]},"commandId":"49d29055-d127-4f0b-87bf-c12f2796ed57","partitions":{"f3d306d9-79ac-4087-afbc-46f739226eb2":{"eventId":12,"partitionAuthorityURI":"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest"}}}],"type":"TRANSACTED","version":"0.1","timeStamp":1534425326898,"commandId":"7a09fb40-199e-4355-84c5-79e2bf67e4d8","partitions":{"f3d306d9-79ac-4087-afbc-46f739226eb2":{"eventId":12,"partitionAuthorityURI":"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest"}}},{"actions":[{"typeName":"Blob","id":["bde58637d88a92d60f054d909a7c574577006d0ce76e5be1786e7bc9f214a8d9fcfd629182fb6eb17911660cbd6fd9367ba3dbb9133e785837fe9ff4cc0be06b"],"type":"CREATED","commandId":"bde58637d88a92d60f054d909a7c574577006d0ce76e5be1786e7bc9f214a8d9fcfd629182fb6eb17911660cbd6fd9367ba3dbb9133e785837fe9ff4cc0be06b"},{"id":["a50b0d93-76d8-4f76-8493-2797c63271f9",null,null,"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest?id=f3d306d9-79ac-4087-afbc-46f739226eb2"],"typeName":"Media","type":"FIELDS_SET","sets":{"content":["bde58637d88a92d60f054d909a7c574577006d0ce76e5be1786e7bc9f214a8d9fcfd629182fb6eb17911660cbd6fd9367ba3dbb9133e785837fe9ff4cc0be06b"]},"commandId":"208037a5-74d8-4d8b-8380-9796329f39da","partitions":{"f3d306d9-79ac-4087-afbc-46f739226eb2":{"eventId":13,"partitionAuthorityURI":"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest"}}}],"type":"TRANSACTED","version":"0.1","timeStamp":1534425366454,"commandId":"2798217b-9b80-4984-a743-4af4387dd71d","partitions":{"f3d306d9-79ac-4087-afbc-46f739226eb2":{"eventId":13,"partitionAuthorityURI":"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest"}}},{"actions":[{"typeName":"Blob","id":["a32802871f32b103bdeee5f88ee2c8e14bbac4e66a6c4f2f796672ab2141e63721f1d4a9dcc04056642e33bd781722f43b97931b5d38bcf670624d14a59cc464"],"type":"CREATED","commandId":"a32802871f32b103bdeee5f88ee2c8e14bbac4e66a6c4f2f796672ab2141e63721f1d4a9dcc04056642e33bd781722f43b97931b5d38bcf670624d14a59cc464"},{"id":["ad61a966-fa8f-495c-bd3e-52521d640d43",null,null,"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest?id=f3d306d9-79ac-4087-afbc-46f739226eb2"],"typeName":"Media","type":"FIELDS_SET","sets":{"content":["a32802871f32b103bdeee5f88ee2c8e14bbac4e66a6c4f2f796672ab2141e63721f1d4a9dcc04056642e33bd781722f43b97931b5d38bcf670624d14a59cc464"]},"commandId":"b839369c-fe36-4854-8bc3-c0450a6300d2","partitions":{"f3d306d9-79ac-4087-afbc-46f739226eb2":{"eventId":14,"partitionAuthorityURI":"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest"}}}],"type":"TRANSACTED","version":"0.1","timeStamp":1534425404902,"commandId":"6ed34c5c-e76b-431a-bd16-21a0f65f3c82","partitions":{"f3d306d9-79ac-4087-afbc-46f739226eb2":{"eventId":14,"partitionAuthorityURI":"valaa-aws://j44low9hwa.execute-api.eu-west-1.amazonaws.com/developtest"}}}]}
```

Comment:
Same as with the other json: the cloud URI's need to be replaced with `valaa-local:`